src/utils/IntlWrapper.ts

```diff
@@ -26,139 +28,111 @@ const dateUnitToMs = {
   seconds: 1000,
 } as const;
 
-enum Rules {
-  DurationFormat,
-  Progress,
-  Range,
-  DateTime,
-}
+export class IntlDuration {
+  protected duration?: durationFormat;
 
-export class IntlWrapper {
-  // Output variables
-  protected output: {
-    text: string;
-    progress: {
-      time?: durationFormat;
-      isFuture: boolean;
-    };
-  } = {
-    text: '',
-    progress: {
-      isFuture: false,
-    },
-  };
+  protected relativeTime?: number;
 
-  // Input variables
-  protected from: Date | number = new Date();
+  protected isFuture = false;
 
-  protected to: Date | number = new Date();
+  protected isNow = false;
```

Comment:
I mean isNow only working for "Progress" rule, for events and other stuff you should use different rules no?

Reply:
Ok, now I get it, since it's better to have "relative time" relative to specific date and not just "now" we move it to DateTime class since you can specify date here.